Makefile

```diff
@@ -114,13 +114,31 @@ test: format
 	@echo "--> Running go test" ;
 	@go test $(UNIT_TEST_PACKAGES)
 
+
+.PHONY: deps
+deps:
+	@echo "--> Tidying up submodules"
+	@go mod tidy
+	@echo "--> Verifying submodules"
+	@go mod verify
+
+.PHONY: verify-deps
+verify-deps: deps
```

Comment:
question: @prateekpandey14  why is this check required?

Reply:
to validate, if we missed to commit a updated mod files